### ISSUE TITLE
Clicking an HxDropdownItemNavLink with an href does not trigger navigation

### ISSUE THREAD

**Issue by skruis** [havit/Havit.Blazor#168]
Hello! Using Edge to run Blazor WASM .NET 6 with version 2.5.1. I'm a new user of these components. I expected that populating the href and then clicking the item would be enough to trigger navigation. It's possible that I'm missing something. Here's my markup:

```
<HxDropdown>
	<HxDropdownToggleElement ElementName="a" CssClass="btn bg-light" role="button">
		<HxIcon Icon="BootstrapIcon.PersonCircle" CssClass="text-primary" />
	</HxDropdownToggleElement>
	<HxDropdownMenu CssClass="dropdown-menu-end">
		<HxDropdownItemNavLink Href="Account">My Account</HxDropdownItemNavLink>
		<HxDropdownDivider />
		<HxDropdownItemNavLink Href="Logout" Match="NavLinkMatch.Prefix">Logout</HxDropdownItemNavLink>
		<HxDropdownItem OnClick="@(() => _nav.NavigateTo("/logout"))">Logout 2</HxDropdownItem>
	</HxDropdownMenu>
</HxDropdown>
```

The Logout 2 link using the OnClick is working. Is there something I'm missing with regards to the href?

Thanks!

-Sean

**Comment by hakenr** [comment]
Reproduced, our demo itself has this bug:
https://havit.blazor.eu/components/HxDropdownButtonGroup - Link with Href

**Comment by hakenr** [comment]
fixed, will be part of next release (v2.6.1)
